source/localizable/smart_cart/webhook.html.md.erb

```diff
@@ -113,6 +113,67 @@ Name           | Type   | Value | Description
 
 <%= partial 'localizable/smart_cart/order_object' %>
 
+## Test requests (sandbox)
+
+For [demo orders](/smart_cart/orders_api/#test-requests-sandbox), separate endpoints are available to trigger different
```

Comment:
minor: "separate endpoints" > "separate API endpoints". 

Also maybe make "API endpoints" and "trigger different kinds of Webhook requests" bold?